models/contact/src/index.ts

```diff
@@ -189,17 +189,6 @@ export function createModel (builder: Builder): void {
     presenter: contact.component.ChannelsPresenter
   })
 
-  builder.createDoc(
```

Comment:
Not sure it is a good idea, since it could be many email providers.

Reply:
I think when we will have several providers then we will be puzzled by this, because there will be additional questions on the logic of choosing a provider that will have to be resolved